includes/REST_API/Stories_Media_Controller.php

```diff
@@ -175,37 +173,4 @@ protected function prepare_items_query( $prepared_args = [], $request = null ) {
 	protected function get_media_types() {
 		return $this->get_allowed_mime_types();
 	}
-
-	/**
-	 * Filters the current query to hide all automatically extracted poster image attachments.
-	 *
-	 * Reduces unnecessary noise in the media library.
-	 *
-	 * @since 1.0.0
-	 *
-	 * @param \WP_Query $query WP_Query instance, passed by reference.
-	 *
-	 * @return void
-	 */
-	public function filter_poster_attachments( &$query ) {
```

Comment:
Don't you have to change or remove this - https://github.com/google/web-stories-wp/blob/49adc48243db2d149c8623119ae63e838bbe971d/tests/phpunit/tests/REST_API/Stories_Media_Controller.php#L216-L244